brainstormforce/astra
# AST-3432 - Fix: Issue with WooCommerce Single Product Layout not applying correctly.

### Description

- WooCommerce Single Product page layout not applying correctly as expected when all WooCommerce Settings set to default.
- Also added Content Style and Sidebar Style support in few places where it is supposed to be added.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3432

### Screenshots

https://d.pr/v/DbHEHI

### Types of changes




Bug Fix

### How has this been tested?

- Check the Default Cases for Single Product.
- Check the Meta Cases for Single Product.
- Check other layouts for Single Product.
- Check the migration cases for Single Product.
- Check other pages like WooCommerce Shop etc to be sure.
- Check Content Color and Site Color settings along with backward cases.

### Checklist:
